app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :yard_sales
+  has_one :saved_yardsale
```

Comment:
Shouldn't this be has_many? Because a user can save many yard sales?

Reply:
has been updated